packages/components/src/template-registry.ts

```diff
@@ -27,6 +30,19 @@ import type HdsLinkToModelsHelper from './helpers/hds-link-to-models';
 import type HdsLinkToQueryHelper from './helpers/hds-link-to-query';
 
 export default interface HdsComponentsRegistry {
+  // Accordion
+  'Hds::Accordion': typeof HdsAccordionIndexComponent;
+  'hds/accordion': typeof HdsAccordionIndexComponent;
+  HdsAccordion: typeof HdsAccordionIndexComponent;
+
+  'Hds::Accordion::Item': typeof HdsAccordionItemButtonComponent;
+  'hds/accordion/item': typeof HdsAccordionItemButtonComponent;
+  HdsAccordionItem: typeof HdsAccordionItemIndexComponent;
```

Comment:
@alex-ju the second side of the declarations should be `ItemIndex`, no?
```suggestion
  'Hds::Accordion::Item': typeof HdsAccordionItemIndexComponent;
  'hds/accordion/item': typeof HdsAccordionItemIndexComponent;
  HdsAccordionItem: typeof HdsAccordionItemIndexComponent;
```

Reply:
(btw, this `template-registry` process is **so** error prone! I have to double check everything twice, because is so much easy to make a typo and not notice it)

Reply:
good catch! yeah, ideally we would have this automated

Reply:
Addressed in [3775a5a](https://github.com/hashicorp/design-system/pull/2101/commits/3775a5adc609b74d069e056da20ede9f42fafbe8)